model.py

```diff
@@ -97,10 +97,13 @@ def forward(self, us, them, w_in, b_in):
   def step_(self, batch, batch_idx, loss_type):
     us, them, white, black, outcome, score = batch
 
-    q = self(us, them, white, black)
-    t = outcome
-    # Divide score by 600.0 to match the expected NNUE scaling factor
-    p = (score / 600.0).sigmoid()
+    # 600.0 to match the expected NNUE scaling factor
+    nnue2score = 600
```

Comment:
could we perhaps rename this, or at least document, that this is `kPonanzaConstant` from the nodchip trainer?

Reply:
like:
```# 600.0 is the kPonanzaConstant scaling factore needed to convert the training net output to a score```
?

Reply:
that and that it needs to match in stockfish

Reply:
in stockfish, or in the serializer ?

Reply:
Right, sorry, in the serializer. Stockfish doesn't need it obviously because it's baked into the weights.